src/tidecli/api/routes.py

```diff
@@ -197,3 +199,8 @@ def submit_task(
         raise click.ClickException("No feedback received")
 
     return TimFeedback(**feedback)
+
+
+def get_task_points(ide_task_id: str, doc_path: str) -> PointsData:
+    res = tim_request(endpoint=TASK_POINTS_ENDPOINT, method="GET", params={"ide_task_id": ide_task_id, "doc_path": doc_path})
+    return PointsData(**res)
```

Comment:
This throws a Pydantic error if the task doesn't give current points (no points specified or no submissions). Could be fixed on TIM side as well.

Reply:
Fixed by allowing None value for current points in cli